go/runtime/registry/config/config.go

```diff
@@ -0,0 +1,142 @@
+// Package config implements global configuration options.
+package config
+
+import (
+	"fmt"
+	"time"
+)
+
+// RuntimeProvisioner is the runtime provisioner.
+type RuntimeProvisioner string
+
+const (
+	// RuntimeProvisionerMock is the name of the mock runtime provisioner.
+	//
+	// Use of this provisioner is only allowed if DebugDontBlameOasis flag is set.
+	RuntimeProvisionerMock = "mock"
```

Comment:
FYI, this is not actually using the `RuntimeProvisioner` type (also below).

Reply:
Yeah, this is a copy&paste of the existing registry code -- I'll fix it :)